test/locater_test.js

```diff
@@ -56,8 +73,8 @@ describe("locater", function() {
     });
 
     it("matches multiple occurences of a regex in a line", function() {
-      var result = locater.find(/[a-zA-Z]{6}/g, 'kaskade - 4AM');
```

Comment:
I think `find` here was meant to match twice: `kaskad` and `askade`. This was passing in `master`.